src/pudl/analysis/plant_parts_eia.py

```diff
@@ -1768,3 +1495,87 @@ def assign_record_id_eia(test_df: pd.DataFrame, plant_part_col: str = "plant_par
         )
     test_df_ids = pd.concat(dfs)
     return test_df_ids
```

Comment:
Ugh I can't leave this comment higher at the function header but do we still need this `assign_record_id_eia` helper function? I don't really understand the Codecov reports but I think it might be complaining about this function because it's not actually used anywhere anymore.

Reply:
I don't think we need it anymore! the underlying `add_record_id` does the bulk of the work here and `assign_record_id_eia` was previously being used in the previous true gran stuff. so.. chuck it!

Reply:
chucked!